backend/elearning/settings.py

```diff
@@ -37,6 +37,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'learnings'
```

Comment:
separate the APPS in one of these ways

note: they are sorted alphabetically
```
INSTALLED_APPS = [
    # django apps
    django apps

    # third party packages
    third party packages

    # local apps
    local apps
```
or
``` 
DJANGO_APPS = [django apps]
THIRD_PARTY_PACJKAGES = [third part packages]
LOCAL_APPS = [local apps]

INSTALLED_APPS = DJANGO_APPS + THYIRD_PARTY_PACKAGES  + LOCAL_APPS
```

Reply:
Please do not include the secret key in the repository.
put it in `.env` file then add the .env file in `.gitignore`
then create a `.env.example` file, which will be included in the repository